Gemfile.lock

```diff
@@ -1,104 +1,103 @@
+PATH
+  remote: .
+  specs:
+    dap (1.3.0)
```

Comment:
```suggestion
    dap (1.3.1)
```

Since this PR bumps the version this needs to match it so a `bundle install` on fresh checkout will not modify `Gemfile.lock`.